src/test/java/io/vertx/core/http/HttpTLSTest.java

```diff
@@ -921,13 +922,15 @@ public TrustOptions copy() {
     boolean serverOpenSSL;
     boolean serverUsesAlpn;
     boolean serverSSL = true;
+    boolean proxyProtocolSupport = false;
```

Comment:
actually we should rename to `useProxyProtocol` like we do have `useAlpn` and `useProxyAuth` already